dataactvalidator/scripts/load_cfda_data.py

```diff
@@ -0,0 +1,100 @@
+
+logger = logging.getLogger(__name__)
+
+
+def load_cfda_program(base_path):
+    """
+    Load cfda program.
+    Parameters
+    ----------
+        base_path : directory that contains the cfda values files.
+    """
+    if CONFIG_BROKER["use_aws"]:
+        s3connection = boto.s3.connect_to_region(CONFIG_BROKER['aws_region'])
+        s3bucket = s3connection.lookup(CONFIG_BROKER['sf_133_bucket'])
+        filename = s3bucket.get_key("cfda_program.csv").generate_url(expires_in=600)
+    else:
+        filename = os.path.join(base_path, "cfda_program.csv")
+
+    logger.info('Loading CFDA program file: ' + "cfda_program.csv")
+    """Load country code lookup table."""
+    model = CFDAProgram
+
+    with create_app().app_context():
+        configure_logging()
+        sess = GlobalDB.db().session
+        # for object class, delete and replace values
+        sess.query(model).delete()
+
+        data = pd.read_csv(filename, dtype=str, encoding='latin1')
+
+        data = clean_data(
+            data,
+            model,
+            {"program_title": "program_title",
+             "program_number": "program_number",
+             "popular_name_(020)": "popular_name",
+             "federal_agency_(030)": "federal_agency",
+             "authorization_(040)": "authorization",
+             "objectives_(050)": "objectives",
+             "types_of_assistance_(060)": "types_of_assistance",
+             "uses_and_use_restrictions_(070)": "uses_and_use_restrictions",
+             "applicant_eligibility_(081)": "applicant_eligibility",
+             "beneficiary_eligibility_(082)": "beneficiary_eligibility",
+             "credentials/documentation_(083)": "credentials_documentation",
+             "preapplication_coordination_(091)": "preapplication_coordination",
+             "application_procedures_(092)": "application_procedures",
+             "award_procedure_(093)": "award_procedure",
+             "deadlines_(094)": "deadlines",
+             "range_of_approval/disapproval_time_(095)": "range_of_approval_disapproval_time",
+             "appeals_(096)": "appeals",
+             "renewals_(097)": "renewals",
+             "formula_and_matching_requirements_(101)": "formula_and_matching_requirements",
+             "length_and_time_phasing_of_assistance_(102)": "length_and_time_phasing_of_assistance",
+             "reports_(111)": "reports",
+             "audits_(112)": "audits",
+             "records_(113)": "records",
+             "account_identification_(121)": "account_identification",
+             "obligations_(122)": "obligations",
+             "range_and_average_of_financial_assistance_(123)": "range_and_average_of_financial_assistance",
+             "program_accomplishments_(130)": "program_accomplishments",
+             "regulations__guidelines__and_literature_(140)": "regulations_guidelines_and_literature",
+             "regional_or_local_office_(151)": "regional_or_local_office",
```

Comment:
Some bugs were addressed/fixed in a recently merged PR:
https://github.com/fedspendingtransparency/data-act-broker-backend/pull/1240

Reply:
Yes. You'll need to be sure to merge that branch down (from wherever those fixes were made -- make sure they have been merged up to the target branch you're merging into). Then resolve any merge conflicts, from having removed the code that was edited/fixed, then re-copy the fixed code into your new file. (or do the same in a shorter way -- e.g. just copy the code from that branch and paste it into your file).